pages/06.policies/09.Promoting-integrations/docs.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Policy on promoting integrations with Mautic'
+twitterenable: true
+twittercardoptions: summary
+articleenabled: false
+orga: {  }
+personenabled: false
+facebookenable: true
+menu: 'Promoting integrations'
+taxonomy:
+    category:
+        - docs
+---
+
+# Introduction
+
+The Mautic website serves as a hub for the Mautic community, showcasing innovative projects that align with our mission and values. The purpose of this policy is to provide clarity on the criteria and process for listing projects and integrations on the Mautic website's home page.
+
+# Criteria for Inclusion
+
+Projects featured on the Mautic website home page are selected based on their popularity and relevance to Mautic users. The decision to list a project is primarily made by the marketing team, considering factors such as the project's impact, use cases, and overall community value.
+
+1. Popularity:
+
+- Projects that have gained popularity within the Mautic community are more likely to be featured on the home page.
+
+- Popularity may be determined by factors such as community engagement, user feedback, and adoption rates.
```

Comment:
```suggestion
```

Reply:
This is far too subjective and open to bias. I think we should instead allow the community to direct this.